docs/changes.rst

```diff
@@ -545,6 +545,9 @@ Compatibility notes:
   unexpected) was rather confusing and unexplainable at first to users.
   If you want that "do not backup NODUMP-flagged files" behaviour, use:
   borg create --exclude-nodump ...
+- If you are on Linux and have many small files, consider using ``--nobsdflags``
+  with ``borg create`` to avoid additional syscalls and speed up backup creation.
+  Not all Linux filesystems fully support BSD flags so you may not need them.
```

Comment:
I'ld kill that last sentence.

It does not matter whether bsdflags are fully supported by the fs, but rather whether the user uses the ones which are supported and wants to have them in the backup archive.